### ISSUE TITLE
update random-access-rn-file

### ISSUE THREAD

**Issue by chrmod** [cliqz/dat-react-native#3]
Current version does not work anymore due to changes to react-native-fs. The proper full path to file has to be provided:
https://github.com/cliqz/dat-react-native/blob/3e3e6904da5bf3933439c534b2659822e28788fc/storage.js#L6

Additionally a support for folders would be nice:
https://github.com/cliqz/dat-react-native/blob/3e3e6904da5bf3933439c534b2659822e28788fc/storage.js#L27

**Comment by chrmod** [comment]
@RangerMauve maybe you can take a look :-)

**Comment by RangerMauve** [comment]
Oof, I'd strongly consider migrating to use nodejs-mobile or something similar. The pefromance hit of serializing and deserializing the FS calls was painfully slow from when @allain and I were looking at it a while ago. 😅

**Comment by martinheidegger** [comment]
With turbo-modules this should become a lot faster, I do have some other thoughts on how to improve this.

**Comment by RangerMauve** [comment]
Yeah, if you can get the performance working decently with RN the UX would be way nicer for app authors.
